src/views/user/EditProfilePage.jsx

```diff
@@ -0,0 +1,143 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link, withRouter } from "react-router-dom"
+import validator from 'validator';
+import PropTypes from 'prop-types';
+import updateProfile from '../../actions/updateProfile';
+import EditProfile from '../../components/EditProfile.jsx';
+
+class EditProfilePage extends Component {
+  constructor() {
+    super();
+    this.handleChange = this.handleChange.bind(this);
+    this.mouseOutHandler = this.mouseOutHandler.bind(this);
+    this.handleUpdateProfile = this.handleUpdateProfile.bind(this);
+    this.state = {};
+  }
+
+
+
+  handleChange(event) {
+    event.persist();
+    const { name, value, id } = event.target;
+    if((name === 'firstName' || name === 'lastName' || name === 'email'|| name === 'bio')
```

Comment:
Please comment this code to provide more clarity on what you are trying to achieve